metricflow/sql/sql_plan.py

```diff
@@ -102,6 +102,25 @@ class SqlSelectColumn:
     # Always require a column alias for simplicity.
     column_alias: str
 
+    @staticmethod
+    def from_table_and_column_names(table_alias: str, column_name: str) -> SqlSelectColumn:
+        """Create a column that selects a column from a table by name."""
+        return SqlSelectColumn(
+            expr=SqlColumnReferenceExpression.from_table_and_column_names(
+                column_name=column_name, table_alias=table_alias
+            ),
+            column_alias=column_name,
+        )
+
+    def ref_with_new_table_alias(self, new_table_alias: str) -> SqlColumnReferenceExpression:
```

Comment:
It was a little unclear what this meant initially. How about something like:

```
def reference_from(self, source_table_alias: str) -> SqlColumnReferenceExpression:
```

Or `source_column_reference`